src/main/scala/com/mwz/sonar/scala/ScalaPlugin.scala

```diff
@@ -104,9 +111,9 @@ object Scala {
       .toList
 }
 
-/** Plugin entry point */
+/** Sonar Scala plugin entry point */
 final class ScalaPlugin extends Plugin {
-  override def define(context: Plugin.Context): Unit = {
+  override def define(context: Context): Unit = {
```

Comment:
IMHO, Since sonar defines context for almost every point of extension, we should stick in always using the relative type. Thus, in this case `Plugin.Context`.

Reply:
Yes Yes, I'm aware of that.
I'm saying that, in my opinion, it is cleaner to use `Plugin.Context`. And standard, since we've been doing that way in all the code base.

Reply:
ok, sure, I'm happy to change it back.

Reply:
done in 2ef18d2